ReadMe.md

```diff
@@ -64,7 +65,7 @@ To display | 表示（する） | ひょうじ |
 
 English | Japanese | Kana | Notes
 --------|----------|------|------
-Device | 端末 | たんまつ | Usually smartphones and tablets
+Device, terminal | 端末 | たんまつ | Often used to mean smartphones and tablets, but can also mean command-line
```

Comment:
Please try to keep it short, eg.: `Usually smartphones and tablets, also command-line.`